### DISCUSSION TITLE
Does sending a message to a group work?

### DISCUSSION THREAD

**Discussion by ork44** [devlikeapro/waha#60] [Q&A] [answered]
Im trying to send a message to a whatsapp group but it returns an 500 error.

Im using the group id + @g.us but its not working.

**Comment by allburov** [comment]
Hi! Yes, it should work.

1.  Where yiu get the group id?
2. Could you attach the 500 response here please with version and the error message

**Reply by ork44** [reply]
I got the group id from taking the last part of the invite url. 

and the 500 error is:
<img width="513" alt="Screenshot 2023-02-14 at 17 52 13" src="https://user-images.githubusercontent.com/44115929/218803753-20b091d9-e5d7-4141-8297-3b4c9b72f783.png">

**Comment by allburov** [comment, marked as answer]
I'm bot sure that it's the right id. 
Could you tey to use webhook to get the id as described here? https://github.com/devlikeapro/whatsapp-http-api/issues/43#issuecomment-1378191600

**Reply by ork44** [reply]
Yes that worked, thank you so much!
